rest-api/src/test/java/io/quarkus/qe/RepositoryResourceTest.java

```diff
@@ -88,10 +99,19 @@ public void shouldReturnConflictIfRepositoryAlreadyExists() {
         thenResponseErrorMessageIs(EXPECTED_CONFLICT_ERROR_MSG);
     }
 
+    @Test
+    public void shouldReturnRepositoryNotFound() throws RepositoryNotFoundException {
+        givenExistingRepository(REPO_URL);
+        whenUpdateRepository(NOT_FOUND_ENTITY_ID);
```

Comment:
Why do we need to create a repository to cover this? If we don't call to the givenExistingRepository, then the repository won't exist ever.